src/amo/components/Link.js

```diff
@@ -18,7 +19,12 @@ export class LinkBase extends React.Component {
   }
 
   render() {
-    const { base, children, to } = this.props;
+    const { base, children, href, to } = this.props;
+
+    if (typeof href === 'string' && href.startsWith('/')) {
+      const linkHref = path.join(base, href);
+      return <Link {...this.props} href={linkHref}>{children}</Link>;
```

Comment:
Oh, sorry! I thought the new line was exactly the same as [the old line](https://github.com/mozilla/addons-frontend/blob/69501369755923cc1e7f8afe67b28941d60d0fb6/src/amo/components/Link.js#L40) but now I see that they use different attributes.

But now I'm totally confused because I don't see the `href` property documented for [`Link`](https://github.com/ReactTraining/react-router/blob/v2.8.1/docs/API.md#link). What does it do?